### PR TITLE
fixed notifications if the user's machine was powered on after the scheduled time had past

### PR DESCRIPTION
**Pull request by QueTeddy**
Added notifications

### CODE REVIEW

File: leeteasy/__main__.py [hunk 1]
```diff
@@ -31,6 +32,10 @@ def execute_start(time, difficulty, sleep_duration) -> None:
     Example: leeteasy 13:15
     """
     TimeValidator.validate(time)
```
Comment (sudiptob2) [inline leeteasy/__main__.py]:
```suggestion
    valid_time = TimeValidator.validate(time)
```

File: leeteasy/__main__.py [hunk 2]
```diff
@@ -31,6 +32,10 @@ def execute_start(time, difficulty, sleep_duration) -> None:
...
+
+    if datetime.now().time() > TimeValidator.validate(time):    #To notify users of missed notifications
```
Comment (sudiptob2) [inline leeteasy/__main__.py]:
```suggestion
    if datetime.now().time() > valid_time:
```

**Review by sudiptob2 — CHANGES_REQUESTED** [review]
I have suggested some changes :pray: 
Also, please rename your branch and add proper commit messages according to the [contributing guidelines](https://github.com/sudiptob2/leet-easy/blob/main/docs/CONTRIBUTING.md)

File: leeteasy/__main__.py (hunk 2 above)
Comment (QueTeddy) [reply leeteasy/__main__.py]: I have added the requested fixes but what do I rename my branch to?
Can't seem to think of a decent name

**Review by sudiptob2 — APPROVED** [review]
seems fine, can you run the `.github/check.sh` script and confirm that it doesn't give any code quality error? :pray:

**Comment by QueTeddy** [comment]
I ran the check and it came out clean. Could I ask what the check.sh script does and how?

Could you point me to any documentation that might help? Thanks

**Comment by sudiptob2** [comment]
> I ran the check and it came out clean. Could I ask what the check.sh script does and how?
> 
> Could you point me to any documentation that might help? Thanks

Great, it actually checks several things, 

1.  run unit tests
2.  `flake8` checks
3.  `mypy` checks

Actually now, I asked you to run it manually but I will set up a **GitHub action** so that when anyone create a **PR** it will run automatically by GitHub, and merge will be blocked if there are any errors.

**Comment by QueTeddy** [comment]
Okay thanks, I would read more on the type of checking as I could see static and dynamic. 

Could you merge the PR?

**Comment by sudiptob2** [comment]
> Okay thanks, I would read more on the type of checking as I could see static and dynamic.
> 
> Could you merge the PR?

will merge it shortly

**Comment by QueTeddy** [comment]
Are there any other problems you would like me to have a look at?

The experience was very nice.

**Comment by sudiptob2** [comment]
> 

Thanks :blush: follow me on **github/twitter/linkedin** for future opensouce projects :pray:

**Comment by sudiptob2** [comment]
hi @QueTeddy, it seems that there is some problems in your code and cached by the `chek.sh` script. please fix that :pray:

File: leeteasy/__main__.py [hunk 3]
```diff
@@ -31,9 +31,9 @@ def execute_start(time, difficulty, sleep_duration) -> None:
 
     Example: leeteasy 13:15
     """
-    TimeValidator.validate(time)
+    valid_time = TimeValidator.validate(time)
 
-    if datetime.now().time() > TimeValidator.validate(time):    #To notify users of missed notifications
+    if datetime.now().time() > valid_time:    #To notify users of missed notifications
```
Comment (sudiptob2) [inline leeteasy/__main__.py]:
```suggestion
    if datetime.now().time() > valid_time:
```

**Review by sudiptob2 — CHANGES_REQUESTED** [review]

File: leeteasy/__main__.py [hunk 4]
```diff
@@ -1,6 +1,7 @@
 import os
 import time as clock
 from sys import platform
+import datetime
```
Comment (sudiptob2) [inline leeteasy/__main__.py]:
```suggestion
```

File: leeteasy/__main__.py [hunk 5]
```diff
@@ -1,6 +1,7 @@
 import os
 import time as clock
```
Comment (sudiptob2) [inline leeteasy/__main__.py:2]:
```suggestion
import time as clock
from datetime import datetime
```

**Review by sudiptob2 — APPROVED** [review]

**Pull request merged**